src/integration/java/org/humancellatlas/ingest/file/FileControllerTest.java

```diff
@@ -212,4 +219,28 @@ private void verifyThatValidationStateChangedToDraft(MetadataDocument... values)
             verify(validationStateChangeService, times(1)).changeValidationState(value.getType(), value.getId(), ValidationState.DRAFT);
         });
     }
+
+    @Test
+    public void testValidationJobPatch() throws Exception {
+        //given:
+        File file = new File("test");
+        file = fileRepository.save(file);
+
+        //when:
+        String patch = "{ \"validationJob\": { \"validationReport\": { \"validationState\": \"Valid\" }}}";
+
+        MvcResult result = webApp
+                .perform(patch("/files/{id}", file.getId())
+                        .contentType(APPLICATION_JSON_VALUE)
+                        .content(patch))
+                .andReturn();
```

Comment:
You could use a [result matcher](https://docs.spring.io/spring-framework/docs/current/javadoc-api/org/springframework/test/web/servlet/result/MockMvcResultMatchers.html).
See [web testing spring docs](https://spring.io/guides/gs/testing-web/)
```suggestion
                .andExpect(content().mimeType(MediaType.APPLICATION_JSON));
```

Reply:
This won't work as the expected return type is `application/hal+json;charset=UTF-8`. Personally, I like the current structure as it reads better with the if, when, expect structure of all of our tests

Reply:
Have you seen [ContentResultMatchers](https://docs.spring.io/spring-framework/docs/current/javadoc-api/org/springframework/test/web/servlet/result/ContentResultMatchers.html)?

Reply:
Yes I have but the type we have is not part of MediaType